Docs/articles/Kernel/CGS.md

```diff
@@ -24,6 +24,8 @@ To really draw into the screen we need to use the Canvas class. Let's give a loo
     void DrawPoint(Pen pen, int x, int y) draws a point at the coordinates specified by x and y with the specified pen
     void DrawLine(Pen pen, int x_start, int y_start, int x_end, int y_end) draws a line at the coordinates specified by x_start, y_start and x_end, y_end with the specified pen
     void DrawRectangle(Pen pen, int x_start, int y_start,int width, int height) draws a rectangle specified by a coordinate pair, a width, and a height with the specified pen
+    void DrawImage(Image image, int x, int y) draws nan image at the x and y specified
```

Comment:
Typo nan -> an

Reply:
Also mention the Display method

Reply:
The Display method is for Double Buffering VBE right?

Reply:
yeah it does nothing for the other drivers